CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## 2023-07-24
+### Added
+- Multi Process Worker Support
+- Custom Container Image for Fluentd using OL8-Slim as base Image.
```

Comment:
This is a new image on container-registry right? we should put image tag?

Reply:
Nope, this is referring to custom container images (instructions for creating the custom container images).